app/models/audit/SubmitReportOfReceiptResponseAuditModel.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.audit
+
+import play.api.libs.json.{JsValue, Json}
+
+case class SubmitReportOfReceiptResponseAuditModel(correlationId: String,
+                                                   arc: String,
+                                                   traderId: String,
+                                                   receipt: String) extends AuditModel {
+
+  override val transactionName: String = "report-of-receipt-response"
+
+  override val auditType: String = "ReportOfReceiptResponse"
+
+  override val detail: JsValue = Json.obj(
+      "correlationId" -> correlationId,
+              "ern" -> arc,
+              "traderId" -> traderId,
+              "receipt" -> receipt,
```

Comment:
Minor tweak, use to cause failures although Scala is more relaxed about it now. But might make sense to keep it clean - just in case they become strict on it again.
```suggestion
              "receipt" -> receipt
```